### DISCUSSION TITLE
MiKTeX编译错误：无法启动编译命令

### DISCUSSION THREAD

**Discussion by EvenYeah** [note286/xduts#184] [Q&A] [answered]
报错：
无法启动该命令。
"C:\Program Files\MiKTeX\miktex\bin\x64\xelatex.exe"  -synctex=1 -interaction=nonstopmode %.tex
快速构建的命令改为xelatex+view PDF ，出现这种报错，如何解决？

**Comment by EvenYeah** [comment]
按照readme中，用管理员身份运行，运行两条命令，即可以运行了

**Reply by note286** [reply]
编译文档并不需要管理员权限。

**Comment by EvenYeah** [comment]
我现在还没有配好。下载了miktex，xduugthesis的cls文件，但是编译出错，报错见附件。请问怎么解决呢？




| |
白舒婷20009201368
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年4月17日 09:08 |
| 收件人 | ***@***.***> |
| 抄送人 | BiYao ***@***.***>,
State ***@***.***> |
| 主题 | Re: [note286/xduts] 编译错误：无法启动编译命令 (Discussion #184) |

编译文档并不需要管理员权限。

**Reply by sikouhjw** [reply]
请不要使用邮件回复，看不到你说的附件。

**Reply by note286** [reply]
你不需要下载任何`.cls`文件，TeX Live、MacTeX、MiKTeX等LaTeX发行版已经包含了xduts。

**Comment by note286** [comment, marked as answer]
在这里再次重复一遍，以下内容通过README这个入口都可以获取：

如果选择了MiKTeX，根据README，执行`miktex packages update`来更新当前所有包。

接着，运行`texdoc xduts`来获取手册。

通过阅读手册`xduts.pdf`可以得到如下示例，新建空白文件，例如`main.tex`。

```latex
\documentclass{xduugthesis}
\xdusetup{}
\begin{document}
\chapter{欢迎}
使用\LaTeX{}！
\end{document}
```

根据手册说明，使用XeLaTeX进行编译，即`xelatex main`。此时可在当前目录获取编译生成PDF文件，即`main.pdf`。

**Reply by EvenYeah** [reply]
好的，我配成功了，谢谢！
